packages/main/src/DayPicker.js

```diff
@@ -260,6 +260,9 @@ class DayPicker extends UI5Element {
 			if (weekday < 0) {
 				weekday += 7;
 			}
+
+			const nonWorkingAriaLabel = this._isWeekend(oCalDate) ? "Non-Working Day " : "";
```

Comment:
Shouldn't be this translated as well "Non-Working Day"

Reply:
Done